client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/notifcation-settings/index.tsx

```diff
@@ -0,0 +1,192 @@
+				<div className="notification-settings__content-block">
+					<div className="notification-settings__content-heading">
+						{ translate( 'Notify me about downtime:' ) }
+					</div>
+					<SelectDropdown
+						selectedIcon={
+							<img
+								className="notification-settings__duration-icon"
+								src={ clockIcon }
+								alt="Durations"
+							/>
+						}
+						selectedText={ selectedDuration?.label }
+					>
+						{ durations.map( ( duration ) => (
+							<SelectDropdown.Item
+								key={ duration.time }
+								selected={ duration.time === selectedDuration?.time }
+								onClick={ () => selectDuration( duration ) }
+							>
+								{ duration.label }
+							</SelectDropdown.Item>
+						) ) }
+					</SelectDropdown>
+				</div>
+				<div className="notification-settings__toggle-container">
+					<div className="notification-settings__toggle">
+						<ToggleControl
+							onChange={ setEnablMobileNotification }
+							checked={ enableMobileNotification }
+						/>
+					</div>
+					<div className="notification-settings__toggle-content">
+						<div className="notification-settings__content-heading">{ translate( 'Mobile' ) }</div>
+						<div className="notification-settings__content-sub-heading">
+							{ translate( 'Receive notifications via the {{a}}Jetpack App{{/a}}.', {
+								components: {
+									a: (
+										<a
+											className="notification-settings__link"
+											target="_blank"
+											rel="noreferrer"
+											href={ mobileAppLink }
+										/>
+									),
+								},
+							} ) }
+						</div>
+					</div>
+				</div>
+				<div className="notification-settings__toggle-container">
+					<div className="notification-settings__toggle">
+						<ToggleControl
+							onChange={ setEnableEmailNotification }
+							checked={ enableEmailNotification }
+						/>
+					</div>
+					<div className="notification-settings__toggle-content">
+						<div className="notification-settings__content-heading">{ translate( 'Email' ) }</div>
+						<div className="notification-settings__content-sub-heading">
+							{ translate( 'Receive email notifications with one or more recipients.' ) }
+						</div>
+						{
+							// We are using CSS to hide/show add email content on mobile/large screen view instead of the breakpoint
+							// hook since the 'useMobileBreakpont' hook returns true only when the width is > 480px, and we have some
+							// styles applied using the CSS breakpoint where '@include break-mobile' is true for width > 479px
+						 }
+						<div className="notification-settings__large-screen">{ addEmailsContent }</div>
+					</div>
+				</div>
+				<div className="notification-settings__small-screen">{ addEmailsContent }</div>
+			</div>
+			<div className="notification-settings__footer">
+				<Button
+					onClick={ onClose }
+					aria-label={ translate( 'Cancel and close notification settings popup' ) }
+				>
+					{ translate( 'Cancel' ) }
+				</Button>
```

Comment:
Along with the Cancel button and `X` for closing the modal, can we also dismiss the popup with the Escape key or clicking outside the modal?

Reply:
I purposely added `shouldCloseOnClickOutside={ false }` since this modal is something that has user-filled data, which might get lost if the user clicks outside by mistake. I don't have a very strong opinion on this. 

@madebynoam - any thoughts here?

Reply:
That is another great question. At first, I also thought it would be better to disable Escape, but it seems to be more accessible to allow it.  So let's go ahead and enable it.
https://www.w3.org/WAI/ARIA/apg/example-index/dialog-modal/dialog#accessibilityfeatures
https://clearleft.com/thinking/accessibility-in-modal-dialogs

As for clicking outside, I think it's also ok to allow it (since it's similar to escape).

Reply:
Makes sense. Thanks, @madebynoam :)

Reply:
Looks like it was already getting closed on clicking ESC, I just changed it to close on clicking outside - https://github.com/Automattic/wp-calypso/pull/71373/commits/5ba3e76f146eeb6a6fc20651f313953d74ba45e0